chatdata/config/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "ServiceSettings": {
-        "SiteURL": "http://localhost:8065",
+        "SiteURL": "https://chat.androolloyd.com",
```

Comment:
mmm is this safe for production?

Reply:
doesn't get used in production, is only a development file anyways, will be removed.